src/Admins/types.mo

```diff
@@ -4,4 +4,19 @@ module Admins {
         admins  : [Principal];
     };
 
+    public type Interface = {
```

Comment:
What's the advantage of these interfaces again? I find they just add chore work and make my interpreter yell at me. I find code folding + the main.mo makes things plenty scannable.

Reply:
Yeah, I just want a clear overview of what the module/class is exposing, makes it easier to integrate/write tests imo.
If you find it a chose, I can just remove them.